src/sagemaker/serve/model_server/multi_model_server/server.py

```diff
@@ -0,0 +1,127 @@
+"""Module for the MultiModel Local and Remote servers"""
+from __future__ import absolute_import
+
+import requests
+import logging
+from pathlib import Path
+from sagemaker import Session, fw_utils
+from sagemaker.serve.utils.exceptions import LocalModelInvocationException
+from sagemaker.base_predictor import PredictorBase
+from sagemaker.s3_utils import determine_bucket_and_prefix, parse_s3_url, s3_path_join
+from sagemaker.s3 import S3Uploader
+from sagemaker.local.utils import get_docker_host
+
+MODE_DIR_BINDING = "/opt/ml/model/"
+_DEFAULT_ENV_VARS = {}
+
+logger = logging.getLogger(__name__)
+
+
+class LocalMultiModelServer:
+    """Local Multi Model server instance"""
+
+    def _start_serving(
+        self,
+        client: object,
+        image: str,
+        model_path: str,
+        secret_key: str,
+        env_vars: dict,
+    ):
+        """Placeholder docstring"""
+        self.container = client.containers.run(
+            image,
+            "serve",
```

Comment:
Is this the same for all MMS based containers?

Reply:
This is not specific to model server, and it is defined by the DLC build artifacts of entrypoint script like here https://github.com/aws/deep-learning-containers/blob/55bd56050fa78363d6d38e923e29bae1cff73600/huggingface/build_artifacts/inference/mms-entrypoint.py#L20. We should be good to keep it here.

Reply:
For more information : SageMaker overrides default CMD statements in a container by specifying the serve argument after the image name. The serve argument overrides arguments that you provide with the CMD command in the Dockerfile.

https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-inference-code.html#your-algorithms-inference-code-run-image